vizro-core/src/vizro/static/css/layout.css

```diff
@@ -11,8 +11,7 @@
   display: flex;
   flex: 0 0 auto;
   flex-direction: column;
-  padding: 30px 32px 0 32px;
-  scrollbar-gutter: stable;
+  padding: 40px 32px 0 32px;
```

Comment:
@nadijagraca I think this is an essential principle we should adhere to. If something is misaligned, we shouldn't change the outer container but instead change the margin/padding of the relevant element that is misaligned. The reason is that if we change the outer container, it can affect the alignment of all other elements. For example, because we removed the top-padding here, it looked fine for the navigation, but it was misaligned if there was no navigation or a dashboard title.